portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+      <div class="card">
+        <img
+          class="cardimage"
+          src="./Snapshoot_Portfolio_MultiPostStories.png"
+          alt="multipoststoryimage"
+        />
+        <div class="cardheading">Multi-Post Stories</div>
+        <div class="canopyitemwrap">
+          <div class="cardcanopy">CANOPY</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">Back End Dev</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">2015</div>
+        </div>
+        <div class="cardabout">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <div class="cardtechnology">
+          <div class="technology">html</div>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+    </section>
+    <!-- ////////// Abmout me Section-->
+
+    <section class="about-myselfsection">
+      <div class="about-myself">About Myself</div>
+      <div>
+        <div class="myself-content">
+          <p>
+            I’m a software developer! skilled in React JS, Express JS,
+            javaScript, CSS, HTML, JAVA, Ruby, Ruby on Rails. Hands-on
+            experience in implementing Web Development technologies. Interested
+            in developing softwares. I can help you build a product , feature or
+            website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="aboutconnect-icon">
+          <div class="aboutconnect">LET'S CONNECT</div>
+          <ul class="aboutsocial-icons">
+            <li>
+              <a
+                href="https://twitter.com/amrendrak_"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/amrendraakumar/"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-linkedin-in"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/amrendrakind"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <button type="button" class="btnseeproject">Get my resume</button>
```

Comment:
- [ ] Please, consider following the button styling: Active, Hover, and Pressed (see design):
<img width="757" alt="Screen Shot 2022-02-11 at 18 18 30" src="https://user-images.githubusercontent.com/49064106/153707240-5f885f45-a581-4d5a-9ee0-a8afe4e3e06f.png">